cmd/geth/config.go

```diff
@@ -362,6 +367,13 @@ func quorumValidateConsensus(ethereum *eth.Ethereum, isRaft bool) {
 // the PrivacyEnhancements feature
 func quorumValidatePrivacyEnhancements(ethereum *eth.Ethereum) {
 	privacyEnhancementsBlock := ethereum.BlockChain().Config().PrivacyEnhancementsBlock
+
+	for _, transition := range ethereum.BlockChain().Config().Transitions {
+		if transition.PrivacyPrecompileEnabled {
+			privacyEnhancementsBlock = transition.Block
+		}
+	}
```

Comment:
I think we can avoid many ops as it is one way switch off to on
```suggestion
	if !privacyEnhancementsBlock {
		for _, transition := range ethereum.BlockChain().Config().Transitions {
			if transition.PrivacyPrecompileEnabled {
				privacyEnhancementsBlock = transition.Block
				break
			}
		}
	}
```

Reply:
In theory it's not one way, you could decide to turn it off. Why you would do this I have node idea 😆

Reply:
but here, currently in the code you can't disable
not a pointer

Reply:
🤦 Good point!